### DISCUSSION TITLE
Stu3 or R4

### DISCUSSION THREAD

**Discussion by lordgape** [microsoft/FHIR-Converter#297] [Q&A: Handlebars Engine] [answered]
Please the conversion done by handlebars using templates is it FHIR STU3 or FHIR R4 version?

**Comment by lordgape** [comment]
@ranvijaykumar any help?

**Comment by ginalee-dotcom** [comment, marked as answer]
Hi, sorry for the late response! We currently use FHIR R4 for both handlebars and dotliquid templates.

**Reply by lordgape** [reply]
Thank you @ginalee-dotcom
